scipy/stats/__init__.py

```diff
@@ -453,6 +455,7 @@
 from ._entropy import *
 from ._hypotests import *
 from ._rvs_sampling import rvs_ratio_uniforms, NumericalInverseHermite
+from .unuran import *
```

Comment:
Does this introduce a slow down when loading stats? I am not sure I get the size of this.

Reply:
> Does this introduce a slow down when loading stats?

No noticeable difference locally on my ubuntu. It takes 1 to 1.5 seconds to import stats on master and with UNU.RAN.